context/app/static/js/shared-styles/charts/VerticalStackedBarChart/VerticalStackedBarChart.jsx

```diff
@@ -43,8 +52,15 @@ function VerticalStackedBarChart({
   return (
     <>
       <svg width={parentWidth} height={parentHeight} ref={containerRef}>
-        <GridRows top={margin.top} left={margin.left} scale={yScale} width={xWidth} stroke="black" opacity={0.2} />
-        <Group top={margin.top} left={margin.left}>
+        <GridRows
+          top={updatedMargin.top}
+          left={updatedMargin.left}
+          scale={yScale}
+          width={xWidth}
+          stroke="black"
+          opacity={0.2}
+        />
+        <Group top={updatedMargin.top} left={updatedMargin.left}>
```

Comment:
Maybe factor here and above out into a trivial component?
```
<GridRowsGroup  margin={updatedMargin} scale={yScale} width={xWidth}>
  ...
```

Reply:
Done.